### PR TITLE
rename nfdiv crons

### PR DESCRIPTION
**Pull request by linusnorton**

### CODE REVIEW

File: k8s/namespaces/nfdiv/nfdiv-cron-alert-application-not-reviewed/nfdiv-cron-alert-application-not-reviewed.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  releaseName: nfdiv-cron-system-alert-application-not-reviewed-task
+  releaseName: nfdiv-cron-alert-application-not-reviewed
   chart:
     git: git@github.com:hmcts/nfdiv-cron
     ref: 0.0.7
     path: nfdiv-cron
   values:
     job:
-      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-4da7bf9-20210831134705 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```
Comment (thomast1906) [inline k8s/namespaces/nfdiv/nfdiv-cron-alert-application-not-reviewed/nfdiv-cron-alert-application-not-reviewed.yaml:13]: This PR is changing the image to an older image, can you check this file

File: k8s/namespaces/nfdiv/nfdiv-cron-progress-cases-to-aos-overdue/nfdiv-cron-progress-cases-to-aos-overdue.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  releaseName: nfdiv-cron-system-progress-cases-to-aos-overdue-task
+  releaseName: nfdiv-cron-progress-cases-to-aos-overdue
   chart:
     git: git@github.com:hmcts/nfdiv-cron
     ref: 0.0.7
     path: nfdiv-cron
   values:
     job:
-      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-4da7bf9-20210831134705 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```
Comment (thomast1906) [inline k8s/namespaces/nfdiv/nfdiv-cron-progress-cases-to-aos-overdue/nfdiv-cron-progress-cases-to-aos-overdue.yaml:13]: This PR is changing the image to an older image, can you check this file

File: k8s/namespaces/nfdiv/nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-  releaseName: nfdiv-cron-system-progress-held-cases-task
+  releaseName: nfdiv-cron-progress-held-cases
   chart:
     git: git@github.com:hmcts/nfdiv-cron
     ref: 0.0.7
     path: nfdiv-cron
   values:
     job:
-      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-4da7bf9-20210831134705 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```
Comment (thomast1906) [inline k8s/namespaces/nfdiv/nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml:13]: This PR is changing the image to an older image, can you check this file

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant1-application-approved/nfdiv-cron-remind-applicant1-application-approved.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  name: nfdiv-cron-remind-applicant1-application-approved
+spec:
+  releaseName: nfdiv-cron-remind-applicant1-application-approved
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.7
+    path: nfdiv-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```
Comment (thomast1906) [inline k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant1-application-approved/nfdiv-cron-remind-applicant1-application-approved.yaml:13]: Not the latest image, unless you want to imageAutomation to update this when merged?

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant2/nfdiv-cron-remind-applicant2.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  name: nfdiv-cron-remind-applicant2
+spec:
+  releaseName: nfdiv-cron-remind-applicant2
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.7
+    path: nfdiv-cron
+  values:
+    job:
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```
Comment (thomast1906) [inline k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant2/nfdiv-cron-remind-applicant2.yaml:13]: Not the latest image, unless you want to imageAutomation to update this when merged?

File: k8s/namespaces/nfdiv/nfdiv-cron-alert-application-not-reviewed/nfdiv-cron-alert-application-not-reviewed.yaml (hunk 1 above)
Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-alert-application-not-reviewed/nfdiv-cron-alert-application-not-reviewed.yaml]: updated to latest image but we will also rely on image automation

File: k8s/namespaces/nfdiv/nfdiv-cron-progress-cases-to-aos-overdue/nfdiv-cron-progress-cases-to-aos-overdue.yaml (hunk 2 above)
Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-progress-cases-to-aos-overdue/nfdiv-cron-progress-cases-to-aos-overdue.yaml]: updated to latest image but we will also rely on image automation

File: k8s/namespaces/nfdiv/nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml (hunk 3 above)
Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml]: updated to latest image but we will also rely on image automation

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant1-application-approved/nfdiv-cron-remind-applicant1-application-approved.yaml (hunk 4 above)
Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant1-application-approved/nfdiv-cron-remind-applicant1-application-approved.yaml]: updated to latest image but we will also rely on image automation

File: k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant2/nfdiv-cron-remind-applicant2.yaml (hunk 5 above)
Comment (nitinprabhu) [reply k8s/namespaces/nfdiv/nfdiv-cron-remind-applicant2/nfdiv-cron-remind-applicant2.yaml]: updated to latest image but we will also rely on image automation

**Pull request merged**